app/controllers/inherited_proofing_controller.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'inherited_proofing/mhv/inherited_proofing_verifier'
+require 'inherited_proofing/logingov/service'
+
+class InheritedProofingController < ApplicationController
+  skip_before_action :verify_authenticity_token
```

Comment:
Oh yeah, we can remove this line. This is a CSRF check that the frontend will send back since it's an authenticated route